### PR TITLE
Performance improvements

### PR DESCRIPTION
**Pull request by hedleyroos**
@jbeyers please have a look

### CODE REVIEW

File: composer/models.py [hunk 1]
```diff
@@ -134,9 +178,10 @@ class Row(models.Model):
     @property
     def columns(self):
         """Fetch columns and tiles in a single query"""
+
         # Organize into a structure
         struct = {}
-        tiles = Tile.objects.select_related().filter(
+        tiles = Tile.objects.select_related("column").filter(
```
Comment (jbeyers) [inline composer/models.py:184]: Yeah, what was I thinking here?

**Comment by jbeyers** [comment]
+1, I'm happy with this. Much more efficient.

**Comment by hedleyroos** [comment]
@jbeyers have you merged this or must I?

**Comment by jbeyers** [comment]
Merged and closed.

**Pull request merged**